eventlet/__init__.py

```diff
@@ -2,12 +2,6 @@
 import sys
 import warnings
 
-if sys.version_info < (3, 5):
```

Comment:
Maybe change this to (3, 8), instead of deleting?

Reply:
3.7

Reply:
Done, upgraded to version 3.7. Thanks